marklogic-data-hub/src/main/resources/ml-modules/services/mlJobs.sjs

```diff
@@ -43,7 +43,11 @@ function get(context, params) {
     resp = datahub.jobs.getJobDocsByFlow(flow);
   }
   else if (fn.exists(latest)) {
-    resp = datahub.jobs.getLastestJobDocPerFlow();
+    flowNames = (fn.exists(flowNames)) ? datahub.hubUtils.normalizeToSequence(flowNames) : flowNames;
```

Comment:
I think this REST extension is prime for refactoring. The "get" function is difficult to understand because it supports so many operations, and the error that's thrown - "Incorrect options" - isn't helpful. 

What do you think of starting a new "jobs" service with a "getLatestJobForFlows" endpoint? (I don't think "doc" should be part of the name, that's an implementation detail). I understand if it's too much as I know Mike needs this right away, just throwing it out for consideration, as I do think we want to shift to a set of well-defined endpoints for jobs as soon as possible.